specification/marketplacecatalog/data-plane/Products/preview/2023-01-01-preview/products.json

```diff
@@ -0,0 +1,1818 @@
+          {
+            "type": "string",
+            "name": "api-version",
+            "x-originalName": "apiVersion",
+            "in": "query",
+            "required": true,
+            "description": "Version of the REST API used for the request. "
+          },
+          {
+            "type": "string",
+            "name": "storefront",
+            "in": "query",
+            "description": "Relevant storefront, possible values - 'azure, amp, appsource'. Default='azure'",
+            "default": "azure"
+          },
+          {
+            "type": "string",
+            "name": "language",
+            "in": "query",
+            "description": "Language to search, ISO 639-1 two-letter code, possible values - 'en,cs,de,es,fr,hu,it,ja,ko,nl,pl,pt-br,pt-pt,ru,sv,tr,zh-hans,zh-hant'. Default is \"en\"",
+            "default": "en"
+          },
+          {
+            "type": "string",
+            "name": "market",
+            "in": "query",
+            "description": "Product market value  (the response will include only products that can be sold in the specified market)\nPossible values can be found at https://docs.microsoft.com/en-us/azure/marketplace/marketplace-geo-availability-currencies. Example: 'US'"
+          },
+          {
+            "type": "array",
+            "name": "hideKeys",
+            "in": "query",
+            "description": "Add hide key to retrieve preview items. Hidekeys are provided in 'multi' format: &Hidekeys[]=\"first\"&Hidekeys[]=\"second\"",
+            "collectionFormat": "multi",
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "string",
+            "name": "select",
+            "in": "query",
+            "description": "Selects which properties to include in the results. Example: '$select=displayName'"
+          },
+          {
+            "type": "string",
+            "name": "filter",
+            "in": "query",
+            "description": "Filters the results, based on a Boolean condition. Example: '$filter=productType eq \"VirtualMachine\"'. Fields that can be filtered by are:\n- `displayName`\n- `popularity`\n- `categoryIds`\n- `industryIds`\n- `publisherId`\n- `uniqueProductId`\n- `productType`\n- `operatingSystems`\n- `pricingTypes`\n- `publisherDisplayName`\n- `longSummary`\n- `summary`\n- `linkedAddinsTypes`\n- `description`\n- `supportedProducts`\n- `applicableProducts`\n- `lastModifiedDateTime`\n- `plan.planId`\n- `plan.displayName`\n- `plan.cspState`\n- `plan.altStackReference`\n- `plan.stackType`\n- `plan.categoryIds`\n- `plan.hasProtectedArtifacts`\n- `plan.pricingTypes`\n- `plan.summary`\n- `plan.description`\n- `plan.skuId`\n- `plan.displayRank`\n- `plan.isPrivate`"
+          },
+          {
+            "type": "string",
+            "name": "expand",
+            "in": "query",
+            "description": "Expands related entities inline. Example: '$expand=startingPrice'"
+          },
```

Comment:
I assume "select" and "expand" would be array with collectionFormat=csv ?

Reply:
Also remove the "$" from description.

Reply:
Regarding select and expand, I will add that  "collectionFormat=csv " for clarity.

Reply:
Regarding removing the "$" from the description Will this not be confusing for users as it doesn't accurately describe the odata use case?

Reply:
@eladschartz 

Ref https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options

> ⛔ DO NOT prefix any of these query parameter names with "$" (the convention in the [OData standard](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_QueryingCollections)).

Reply:
Ok. Have removed "$" from descriptions as well